packages/bedrock-components/src/components/sqb-auth-template-switch/sqb-auth-template-switch.feature

```diff
@@ -0,0 +1,48 @@
+@owner:coleton
+@author:coleton
+
+Feature: Switch Templates based on Authentication status
+
+  `sqb-auth-template-switch` provides "logged-in" and "logged-out" templates that are conditionally rendered depending on the the validity of the user identity.
+
+  Scenario: Content in the "logged-in" template are shown when a valid user identity is set.
+    Given there is a valid user identity set
+    And the "logged-in" template has content
+    When the sqb-auth-template-switch is rendered
+    Then the content in the "logged-in" template is rendered in the DOM
+
+  Scenario: Content in the "logged-out" template are shown when the user identity is empty.
+    Given there is no valid user identity set
+    And the "logged-out" template has content
+    When the sqb-auth-template-switch is rendered
+    Then the content in the "logged-out" template is rendered in the DOM
+
+  Scenario Outline: Content not matching status is not rendered in the DOM
+    Given there <isOrNot> a valid user identity set
+    And the "logged-in" template has content
+    And the "logged-out" template has content
+    When the sqb-auth-template-switch is rendered
+    Then the content in the "logged-in" template <loggedInmayBe> rendered in the DOM
+    And the content in the "logged-out" template <loggedOutmayBe> rendered in the DOM
+
+    Examples:
+      | isOrNot | loggedInmayBe | loggedOutmayBe |
+      | is      | is            | is not         |
+      | is not  | is not        | is             |
+
+  Scenario Outline: Templates are switched when the user identity is changed
+    Given there <isOrNot> a valid user identity set
+    And the "logged-in" template has content
+    And the "logged-out" template has content
+    Then the content in the "logged-in" template <loggedInmayBeBefore> rendered in the DOM
+    And the content in the "logged-out" template <loggedOutmayBeBefore> rendered in the DOM
+    When the user identity is changed to <changedUser>
+    Then the content in the "logged-in" template <loggedInmayBeAfter> rendered in the DOM
+    And the content in the "logged-out" template <loggedOutmayBeAfter> rendered in the DOM
+
+    Examples:
+      | isOrNot | loggedInmayBeBefore | loggedOutmayBeBefore | changedUser   | loggedInmayBeAfter | loggedOutmayBeAfter |
+      | is      | is                  | is not               | empty         | is not             | is                  |
+      | is not  | is not              | is                   | a valid value | is                 | is not              |
+
```

Comment:
What happens if the switcher is for some reason missing one or more of its needed child templates? (just want to know how it fails)